server/test/controllers/FileControllerTest.java

```diff
@@ -164,6 +171,21 @@ public void acledAdminShow_globalAdmin_returnsUnauthorizedResult() {
     assertThat(result.status()).isEqualTo(UNAUTHORIZED);
   }
 
+  @Test
+  public void acledAdminShow_globalAdminWithPrivledges_redirects() {
+    when(mockSettingsManifest.getAllowCiviformAdminAccessPrograms(request)).thenReturn(true);
+
+    ProgramModel program = ProgramBuilder.newDraftProgram().build();
+    createProgramAdminWithMockedProfile(program);
+    String fileKey = fakeFileKey(1L, program.id);
+    createStoredFileWithProgramAccess(fileKey, program);
+    String encodedFileKey = encodefakeFileKey(fileKey);
+
+    Result result = controller.acledAdminShow(request, encodedFileKey);
+
+    assertThat(result.status()).isEqualTo(SEE_OTHER);
+  }
+
   @Test
   public void acledAdminShow_redirects() {
```

Comment:
(blocking) It's unclear from the test name if this is now civiform admin or program admin. Based on the code looks like program admin, but can you update the test method name to explicitly state it.

Reply:
Good call! Updated